src/components/pools/list/List.tsx

```diff
@@ -107,8 +108,8 @@ export const List: React.FC<{
       // justifyContent: columns.alignment.network,
     },
     {
-      title: 'Amount in Pool',
-      sortKey: PoolCreated_OrderBy.TotalSupply,
+      title: 'Total deposited',
+      // sortKey: PoolCreated_OrderBy.TotalAmountFunded,
```

Comment:
Should it be commented?

Reply:
This didn't work and still doesn't work. We need to revisit the sort logic to handle different types. But it's out of the scope of this PR imo.